### PR TITLE
Find error in data

### PR DESCRIPTION
**Pull request by antvega**
New service to get different percentages and lists of info in the data. Also fixes a bug with that doesn't handle the case of a rovname with spaces (i.e mini rov)

### CODE REVIEW

**Comment by antvega** [comment]
Also made all endpoints lowercase in all services

File: pom.xml [hunk 1]
```diff
@@ -20,16 +20,39 @@
...
-        <name>bintray</name>
-        <url>https://dl.bintray.com/org-mbari/maven</url>
-      </repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>bintray-org-mbari-maven</id>
+            <name>bintray</name>
+            <url>https://dl.bintray.com/org-mbari/maven</url>
+        </repository>
```
Comment (Martin-52) [inline pom.xml:29]: Align indentation

File: pom.xml [hunk 2]
```diff
@@ -20,16 +20,39 @@
...
+            <id>
+                bintray-org-mbari-maven
+            </id>
+            <name>
+                bintray
+            </name>
+            <url>
+                https://dl.bintray.com/org-mbari/maven
+            </url>
+        </repository> -->
```
Comment (Martin-52) [inline pom.xml:46]: Remove commented code

File: src/main/java/org/mbari/m3/dive/visualizer/DataErrorService.java [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+import mbarix4j.math.Statlib;
+
+
+public class DataErrorService implements Service {
+    private final Logger log = Logger.getLogger(getClass().getName());
+
+    @Override
+    public void update(Routing.Rules rules) {
+
+        // rules.get("/getRovNames", this::getRovNames);
```
Comment (Martin-52) [inline src/main/java/org/mbari/m3/dive/visualizer/DataErrorService.java:44]: Remove

File: src/main/java/org/mbari/m3/dive/visualizer/DataErrorService.java [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+                cameraLogCoverageHttpResponse(req, res, false);
+            } catch (IOException | InterruptedException e) {
+                // TODO Auto-generated catch block
+                e.printStackTrace();
+            } 
+        });
+    }
+
+    /**
+     * Sends http request to retrieve the json for the given rov and dive number
```
Comment (Martin-52) [inline src/main/java/org/mbari/m3/dive/visualizer/DataErrorService.java:101]: I think it'd be beneficial to explain what the json data is in the comments

**Review by Martin-52 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**